src/txmempool.cpp

```diff
@@ -496,12 +496,16 @@ void CTxMemPool::removeUnchecked(txiter it, MemPoolRemovalReason reason)
     // even if not directly reported below.
     uint64_t mempool_sequence = GetAndIncrementSequence();
 
+    const uint256 hash = it->GetTx().GetHash();
```

Comment:
6f40c32fa8dfc742cf5ca7daa28615e066872aec

maybe stop making a copy, and let it be a `Txid`

```suggestion
    const auto& hash = it->GetTx().GetHash();
```

Reply:
👍🏾  Done